test/reporters/csvSearchReporter.test.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import fs from 'node:fs';
+import { TestContext } from '@salesforce/core/testSetup';
+import { expect } from 'chai';
+import { Ux } from '@salesforce/sf-plugins-core';
+import { CsvSearchReporter } from '../../src/reporters/search/csvSearchReporter.js';
+
+describe('CSV search reporter', () => {
+  const $$ = new TestContext();
+
+  beforeEach(() => {});
+
+  afterEach(() => {
+    $$.SANDBOX.restore();
+  });
+
+  it('will write two csv files', () => {
+    const logStub = $$.SANDBOX.stub(Ux.prototype, 'log');
+    const writeFileStub = $$.SANDBOX.stub(fs, 'writeFileSync');
```

Comment:
idea: rather than the fs stubbing, I think this would be better to write to a temp dir (and then use mocha-snap to snapshot and commit them).

Then we'd detect accidental changes (ex: `to.include` would pass if you accidentally included the attributes stuff)